torchrl/envs/vec_env.py

```diff
@@ -87,7 +87,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         del self.dummy_env
 
 
-class _BatchedEnv(_EnvClass):
+class _BatchedEnv(EnvStateful):
```

Comment:
same: stateful? Perhaps it's too hard to make it follow EnvBase in this case. What do you think?